src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/commands.py

```diff
@@ -172,7 +172,7 @@ def load_command_table(self, _):
     # Resource feature commands
     if self.supported_api_version(min_api='2017-05-10'):
         feature_table_transform = '{Name:name, RegistrationState:properties.state}'
-        with self.command_group('feature', resource_feature_sdk) as g:
+        with self.command_group('feature', resource_feature_sdk, client_factory=cf_features) as g:
```

Comment:
Why this change?
If it is needed, I guess you can remove `client_factory=cf_features` below?

Reply:
Yeah, pretty much so I could remove the kwarg in the lines below.